stdlib/list.mli

```diff
@@ -73,6 +73,11 @@ val nth_opt: 'a list -> int -> 'a option
 val rev : 'a list -> 'a list
 (** List reversal. *)
 
+val init : int -> (int -> 'a) -> 'a list
+(** [List.init len f] is [f 0; f 1; ...; f (n-1)].
+
+   Raise [Invalid_argument] if [n < 0].*)
```

Comment:
It is better to be using `@raise Invalid_argument ...` and please do include the `@since 4.05.0` line